.github/workflows/PR_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+This pull request was created by GitHub Actions/AWS CodeBuild! Before merging, please do the following:
+- [ ] Review changelog/staleness report.
+- [ ] Review build/test results by clicking *Build Logs* in CI Report (be patient).
```

Comment:
nit: might be nice to mention that we expect it to take ~4hrs to give ppl an idea